expfmt/encode.go

```diff
@@ -83,6 +133,17 @@ func NewEncoder(w io.Writer, format Format) Encoder {
 			_, err := MetricFamilyToText(w, v)
 			return err
 		})
+	case FmtOpenMetrics:
+		return encoderCloser{
```

Comment:
I'd have all of the above return an encoderCloser too, it'll make it easier to switch in future for users. None of the API usage I saw had an explicit type anywhere.

Reply:
Done.